devicetypes/smartthings/zll-rgb-bulb.src/zll-rgb-bulb.groovy

```diff
@@ -110,9 +109,29 @@ def configure() {
 }
 
 def ping() {
+	unschedule()
+	configureHealthCheck()
 	refreshAttributes()
 }
 
+def configureHealthCheck() {
+	Integer hcIntervalMinutes = 12
+	if (!hasConfiguredHealthCheck) {
```

Comment:
Need to add this to the top of each file: https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy#L14